.github/scripts/update_schutzfile.py

```diff
@@ -0,0 +1,38 @@
+#!/bin/python3
+
+import json
+import os
+import re
+
+repo_files = os.listdir("/repo/")
+singletons = []
+# Get a list of all repositories that contain 'singleton'
+for repo_file in repo_files:
+    with open(os.path.join("repo", repo_file), "r") as f:
+        data = json.load(f)
+    if "singleton" in data.keys():
+        singletons.append(data["snapshot-id"])
+
+with open(os.path.join(os.environ.get("REPO"), "Schutzfile"), "r") as f:
```

Comment:
I think there was a misunderstanding in the last review. I think `REPO` and `SUFFIX` shouldn't be passed inside the script using environmental variables but using command line arguments. A simple `argparse` should be enough. See e.g. https://github.com/osbuild/maintainer-tools/blob/main/update-distgit.py#L103

Reply:
Ah sorry, yeah I didn't understand it correctly. I think I got it now :)